engine/src/main/java/org/terasology/rendering/dag/Node.java

```diff
@@ -26,5 +28,5 @@
 
     void process();
 
-    // TODO: add a "Set getStateChanges();" method here
+    Set<StateChange> getStateChanges();
```

Comment:
In line with what written above this would get renamed into `getDesiredStateChanges();`